CHANGELOG.rst

```diff
@@ -1,5 +1,8 @@
 Changelog for Pulse3D
 =====================
+0.25.0 (2022-08-23)
+- Added the option to set custom y-axis for output graphs
+-------------------
```

Comment:
Sorry last thing, this `-------------------` needs to be moved up one line

Reply:
I will pay attention to the formatting of variables and logs more.